docs/resources/enterprise_project.md

```diff
@@ -0,0 +1,41 @@
+---
+subcategory: "Enterprise Project Management Service (EPS)"
+---
+
+# huaweicloud_enterprise_project
```

Comment:
better to add a `NOTE` to explicitly note that the deletion of this resource will disable it instead of deleting.

Reply:
fixed